app/controllers/comment_controller.php

```diff
@@ -0,0 +1,70 @@
+<?php
+class CommentController extends AppController
+{
+    public function edit()
+    {
+        $comment = new Comment;
+        $page = Param::get('page_next', 'edit');
+        $comment_id = Param::get('id');
+        $comments = Comment::getComment($comment_id);
+        $error = false;
+
+        switch ($page) {
+            case 'edit':
+                break;
+            case 'write':
+                $comment->body = Param::get('body');
+                try {
+                    $comment->edit($comment_id);
+                } catch (RecordNotFoundException $e) {
+                        $page = 'edit';
+                        $error = true;
+                }
+                break;
+            default:
+                throw new NotFoundException("{$page} is not found");
+                break;
+        }
+        $this->set(get_defined_vars());
+        $this->render($page);
+    }
+
+    public function redirect_delete()
```

Comment:
`redirect_delete` is the function name but i think there is no redirect method there?